Remotion/Web/Core/Res/Themes/NovaViso/HTML/TreeView.css

```diff
@@ -174,8 +174,8 @@ ul.treeViewRoot span.treeViewNode span.treeViewNodeBadge
   align-self: start;
 }
 
-ul.treeViewRoot li:has(a:focus) > ul > li:first-of-type span.treeViewNode:not(:hover) span.treeViewNodeBadge,
-ul.treeViewRoot li:has(a:focus) + li span.treeViewNode:not(:hover) span.treeViewNodeBadge
+ul.treeViewRoot li:has(> span.treeViewNode span[class^="treeViewNodeHead"] a[role=treeitem]:focus) > ul > li:first-of-type > span.treeViewNode:not(:hover) span.treeViewNodeBadge,
+ul.treeViewRoot li:has(> span.treeViewNode span[class^="treeViewNodeHead"] a[role=treeitem]:focus) + li:first-of-type > span.treeViewNode:not(:hover) span.treeViewNodeBadge
```

Comment:
since we don't do substring matching, I would prefer to not open pandora's box right now. can we do either two selectors or an OR?

Reply:
I think we should just remove the `span[class^='treeVieNodeHead"]`, it's not really required as we only care about the first `span `being a direct child of `li`

Reply:
oh? I thought it was also thhat the anchor should be the head-anchor not a child-item-anchor?

Reply:
it can't be a child item anchor, the anchor with role=treeitem is unique inside the treeViewNode, so it should be fine.

Reply:
Ah, I checked but did not see :D

Then we can simplify to `> span.treeviewNode a:focus` ?

Reply:
If we think that it's impossible that the <a> expander has focus, then yes, otherwise we have the same problem again of not being specific enough